frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -52,86 +51,112 @@ const getIconClass = (status: ClusterUpdateStatus) => {
   }[status];
 };
 
-const UpdateFailingAlert: React.SFC<UpdateFailingAlertProps> = ({condition, updatesAvailable=false}) => <div className="alert alert-danger">
-  <i className="pficon pficon-error-circle-o" aria-hidden="true" />
-  <strong>Update is failing.</strong>
-  &nbsp;
-  {condition.message && `${condition.message}. `}
-  View <Link to="/settings/cluster/clusteroperators?orderBy=desc&sortBy=Status">Cluster Operators</Link> for more details
-  {updatesAvailable && ' or check other available updates to try another version'}.
-</div>;
+const UpdatesAvailableMessage: React.SFC<CVStatusMessageProps> = ({cv, iconClass}) => <React.Fragment>
+  <div className="co-update-status">
+    {iconClass && <i className={classNames(iconClass, 'update-pending')} aria-hidden={true}></i>}
+    &nbsp;
+    Update available
+  </div>
+  <div>
+    <Button bsStyle="primary" onClick={() => (clusterUpdateModal({cv}))}>
+      Update now
+    </Button>
+  </div>
+</React.Fragment>;
 
-const RetrieveUpdatesFailedAlert: React.SFC<RetrieveUpdatesFailedAlertProps> = ({condition}) => <div className="alert alert-danger">
-  <i className="pficon pficon-error-circle-o" aria-hidden="true" />
-  <strong>Could not retrieve updates.</strong>
-  {condition.message && ` ${condition.message}.`}
-</div>;
+const UpdatingMessage: React.SFC<CVStatusMessageProps> = ({cv, iconClass}) => {
+  const updatingCondition = getClusterVersionCondition(cv, ClusterVersionConditionType.Progressing, K8sResourceConditionStatus.True);
+  return <React.Fragment>
+    <div>
+      {iconClass && <i className={iconClass} aria-hidden={true}></i>}
+      &nbsp;
+      Working towards
+    </div>
+    <div>updatingCondition.message && `${updatingCondition.message}. `</div>
```

Comment:
Don't assume the message doesn't have punctuation already. Only output the `div` if the message exists.

```suggestion
    {updatingCondition.message && <div>{updateCondition.message}</div>}
```